Emergence/Contexts/Presenting Locations/ShowLocationsViewController.swift

```diff
@@ -19,10 +19,9 @@ class ShowLocationsViewController: UIPageViewController, UIPageViewControllerDat
         let network = appVC.context.network
         let coords = location.coordinates()
         let showInfo = ArtsyAPI.RunningShowsNearLocation(amount: 2, lat: coords.lat, long: coords.long)
-        network.request(showInfo).mapSuccessfulHTTPToObjectArray(Show).subscribe(next: { decodables in
-            print("decodables: \(decodables)")
-                self.shows = decodables
+        network.request(showInfo).mapSuccessfulHTTPToObjectArray(Show).subscribe(next: { shows in
```

Comment:
Possible reference cycle with `self` here, if the VC is deallocated before the request completes.